### ISSUE TITLE
Translate Document in Chinese

### ISSUE THREAD

**Issue by MacHu-GWU** [pypa/pipenv#3588]
I can help with writing Chinese documentation and very proficient with Sphinx Doc, just one question, where should I put the chinese .rst files? I saw there's a Japanese version, but I can't find it.

**Comment by techalchemy** [comment]
Hmmm good question, I have actually no idea how this works but most of the maintainers are less ignorant than I am as I think they all speak chinese to some degree

/cc @uranusjr @frostming @jxltom

**Comment by jxltom** [comment]
As https://docs.readthedocs.io/en/latest/localization.html, we might need to do some adjustments to our readthedocs project to support multiple language.

**Comment by frostming** [comment]
Japanese and Spanish translations seem to be done in separate projects(the 1st option in https://docs.readthedocs.io/en/latest/localization.html):

Jap: https://github.com/cocoatomo/pipenv
Es: https://github.com/iKenshu/pipenv

@MacHu-GWU You can create a fork and do CN translation in that repo. We are glad to help.

**Comment by frostming** [comment]
I found a team on [transifex](https://www.transifex.com/cocoatomo/pipenv/). Maybe it is a good place to go ahead. /cc @cocoatomo

**Comment by cocoatomo** [comment]
Hi, I am welcome your application for participation in the Transifex project.
This [CircleCI configuration](https://github.com/cocoatomo/pipenv/blob/701ad0b6e82733fb5790539fc10c4aa6384bdfea/.circleci/config.yml) might helps you to build documentation with translation data.

**Comment by MacHu-GWU** [comment]
@frostming I do can fork and start. Before doing that, may I know the Localized Docs should be included in the main repo, or in a separate repo?

**Comment by MacHu-GWU** [comment]
@cocoatomo I checked your https://github.com/cocoatomo/pipenv, but I can't find your Japanese docs, would you mind to help me locate it?

**Comment by uranusjr** [comment]
@MacHu-GWU The method used is to export strings into the `docs/locales` directory, and translate on Transifex (link above).

**Comment by cocoatomo** [comment]
@MacHu-GWU Sorry for inconvenience, and @uranusjr is right. thanks!
Japanese documentations are published on [readthedocs](https://pipenv-ja.readthedocs.io/ja/translate-ja/).

To understand the features used in my translation project, please refer following resources:

- about internationalization features of Sphinx: http://www.sphinx-doc.org/en/master/usage/advanced/intl.html
- about Transfex client: https://docs.transifex.com/client/introduction

**Comment by frostming** [comment]
@MacHu-GWU I have setup a translation project on Transifex: https://www.transifex.com/pipenv-zh/pipenv-doc. welcome to join!

**Comment by jxltom** [comment]
@frostming I remember transifex supports translation of multiple languages. Could we setup a project such as ```https://www.transifex.com/pipenv``` where we translate pipenv into different language instead of only Chinese?

**Comment by frostming** [comment]
@jxltom Yeah it is great but Japenese and Spanish translation already lie in their own repo.

I can change the org name to `pipenv-tr` since `pipenv` is occupied.

**Comment by jxltom** [comment]
Sounds good! Maybe we could start from Chinese first and then adding other language later.

**Comment by krysal** [comment]
Excuse me @frostming do you know how to contribute to the Spanish docs? I want to translate the last paragraph of this [page](https://pipenv-es.readthedocs.io/es/latest/basics.html) but I don't even know where to start.

**Comment by matteius** [comment]
Seems with the new LLMs and GPT models that translating the docs would have gotten a whole lot easier if that is still something anyone wants to take up.  Also, we should probably finish rewriting/improving the documentation first before making many strides to translate.
